pdns/ws-auth.cc

```diff
@@ -968,7 +968,7 @@ static void apiZoneMetadata(HttpRequest* req, HttpResponse *resp) {
       throw ApiException("Could not update metadata entries for domain '" +
         zonename.toString() + "'");
 
-    DNSSECKeeper::clearMetaCache(zonename);
+    purgeAuthCacheForZone(zonename);
```

Comment:
This flushes a lot more than it used to, is that necessary?

Reply:
"Maybe". Depends on what metadata setting is changed.

If the idea previously was to make DNSSECKeeper see new data, the outcome of that never made it into the packetcache. If its a SOA-EDIT-xxx metadata change, I think its expected that all caches are flushed and queries immediately see the new SOA (edits), etc?